superset-frontend/src/components/Chart/chartAction.js

```diff
@@ -248,7 +248,7 @@ export async function getChartDataRequest({
 
 export function runAnnotationQuery({
   annotation,
-  timeout = 60,
+  timeout = 300,
```

Comment:
Another option is for us to make this timeout configurable in config.py like we do with the server timeouts.

Reply:
+1 on putting this in config such that we can see timeouts in one place and if someone wants to configure it for their instance they just have to update in config.py vs having to find all places and update it there.

Reply:
Yea, would be nice to make this a config :)

Reply:
kk, I'll make that change then. 👍

Reply:
Would it be sensible to use one of the existing timeout configs rather than adding a new one?